zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1142,6 +1142,10 @@ property, when available, is noted below.
     The intent is for the clients not to retry them immediately.
     When set to 0, no requests will be throttled. The default is 0.
 
+* *leader.closeSocketAsync*
```

Comment:
please also mention this parameter in the description of the Quorum TLS config parameter (look for `sslQuorum` in the same zookeeperAdmin.md) as this should be considered when someone is enabling Quorum TLS.

Reply:
done.